ui.py

```diff
@@ -145,6 +146,26 @@ def delete_save():
     return None
 
 
+def rename_save():
+    """ A menu for selecting a save to rename. """
+    save = loop_menu('Rename Save', lambda: (
+        saves_list(lambda s: s) +
```

Comment:
I think that is acceptable, `saves_list` takes a function to return a result for a save, our function returns the name of the save.